jkube-kit/jkube-kit-wildfly-jar/src/test/java/org/eclipse/jkube/wildfly/jar/generator/WildflyJARGeneratorTest.java

```diff
@@ -48,133 +48,132 @@ class WildflyJARGeneratorTest {
+        List<String> extraOptions = generator.getExtraJavaOptions();
+        assertThat(extraOptions).isNotNull()
+                .hasSize(2)
+                .satisfies(option -> assertThat(option).first()
+                        .isEqualTo("-Djava.net.preferIPv4Stack=true"))
+                .satisfies(option -> assertThat(option).last()
+                        .isEqualTo("-Dmaven.repo.local=/deployments/myrepo"));
 
-      List<String> extraOptions = generator.getExtraJavaOptions();
-      assertThat(extraOptions).isNotNull()
-          .hasSize(2)
-          .satisfies(option -> assertThat(option).first()
-              .isEqualTo("-Djava.net.preferIPv4Stack=true"))
-          .satisfies(option -> assertThat(option).last()
-              .isEqualTo("-Dmaven.repo.local=/deployments/myrepo"));
+        List<AssemblyFileSet> fileSets = generator.addAdditionalFiles();
+        assertThat(fileSets).isNotEmpty()
+                .last()
+                .hasFieldOrPropertyWithValue("directory", targetDir.toFile())
+                .extracting(AssemblyFileSet::getIncludes).asList()
+                .singleElement()
+                .isEqualTo("myrepo");
 
-      List<AssemblyFileSet> fileSets = generator.addAdditionalFiles();
-      assertThat(fileSets).isNotEmpty()
-          .last()
-          .hasFieldOrPropertyWithValue("directory", targetDir.toFile())
-          .extracting(AssemblyFileSet::getIncludes).asList()
-          .singleElement()
-          .isEqualTo("myrepo");
     }
     
     @Test
-    void slimServerAbsoluteDir(@Mocked final JavaProject project) throws IOException {
-      Map<String, Object> options = new HashMap<>();
-      Map<String, String> pluginOptions = new HashMap<>();
-
-      Path targetDir = Files.createDirectory(temporaryFolder.resolve("target"));
-      Path repoDir = Files.createDirectory(targetDir.resolve("myrepo"));
-
-      options.put(PLUGIN_OPTIONS, pluginOptions);
-      pluginOptions.put(JBOSS_MAVEN_DIST, null);
-      pluginOptions.put(JBOSS_MAVEN_REPO, repoDir.toString());
-
-      GeneratorContext ctx = contextForSlimServer(project, options, null);
-      WildflyJARGenerator generator = new WildflyJARGenerator(ctx);
-
-      List<String> extraOptions = generator.getExtraJavaOptions();
-      assertThat(extraOptions).isNotNull()
-          .hasSize(2)
-          .satisfies(option -> assertThat(option).first()
-              .isEqualTo("-Djava.net.preferIPv4Stack=true"))
-          .satisfies(option -> assertThat(option).last()
-              .isEqualTo("-Dmaven.repo.local=/deployments/myrepo"));
+    void slimServerAbsoluteDir() throws IOException {
+        Map<String, Object> options = new HashMap<>();
+        Map<String, String> pluginOptions = new HashMap<>();
+        Path tmpDir = Files.createTempDirectory("bootable-jar-test-project2");
+        Path targetDir = tmpDir.resolve("target");
+        Path repoDir = targetDir.resolve("myrepo");
+        Files.createDirectories(repoDir);
```

Comment:
I'm not able to understand the purpose behind these path-related changes. I think if we just stick to mockito statements, test should work.